app/check.php

```diff
@@ -68,7 +68,7 @@
 echo '  than the one used with your web server. To be on the'.PHP_EOL;
 echo '      safe side, please check the requirements from your web'.PHP_EOL;
 echo '      server using the ';
-echo_style('yellow', 'web/check.php');
+echo_style('yellow', 'web/config.php');
```

Comment:
I think this changes should be reverted. We fix it in https://github.com/symfony/symfony-demo/pull/140/files